src/main/java/pl/palonek/interview/course/service/CourseStatusService.java

```diff
@@ -0,0 +1,63 @@
+package pl.palonek.interview.course.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import pl.palonek.interview.course.model.CourseSession;
+import pl.palonek.interview.course.model.CourseSessionDTO;
+import pl.palonek.interview.course.model.SessionStatus;
+import pl.palonek.interview.course.repository.CourseRepository;
+
+import java.time.ZonedDateTime;
+
+
+@Service
+@RequiredArgsConstructor
+public class CourseStatusService {
+
```

Comment:
[minor] All methods return void so there is no way to know what was the result of method execution (although we understand that in a small application it's not a big deal).